authui/src/authflowv2/components/checkbox.css

```diff
@@ -56,8 +50,14 @@
         ring-[length:var(--checkbox\_\_ring-width)]
         ring-[color:var(--checkbox\_\_ring-color)];
 
-      &::before {
-        color: var(--checkbox__check-color);
+      & > .material-icons {
```

Comment:
Instead of getting a specific class, we should introduce a class `checkbox__icon`, or even `checkbox__label`.